webapp/public/javascripts/controllers.js

```diff
@@ -10,6 +10,11 @@ lunchrControllers.controller('MainPageController', ['$scope', '$http', '$locatio
 
         $scope.logIn = function () {
             $scope.errorsMessages = null;
+
+            if(!$scope.email || !$scope.password) {
+                return
```

Comment:
@paymog oh ya sorry I meant to show you earlier. not sure why it didn't show. line 15 and 44